src/com/mostc/pftt/util/HostEnvUtil.java

```diff
@@ -228,11 +219,16 @@ private static void installAndConfigureMySql(FileSystemScenario fs, AHost host,
 			cm.println(EPrintType.IN_PROGRESS, HostEnvUtil.class, "Creating data folder for MySql Server 5.7.25.0");		
 			String data_dir = Dir_Mysql_5_7 + "\\data";
 			createDirectoryIfNotExists(fs, cm, data_dir);
-			
+
+			String name = "MySql-Server-5.7.25";
+
+			// Allow Mysql installer through firewall
+			addRuleToFirewall(cm, host, name, Exe_Mysql_5_7_mysqld);
+
 			// Install the MySQL service
 			cm.println(EPrintType.IN_PROGRESS, HostEnvUtil.class, "Installing MySQL as a windows service");
 			// mysqld.exe --install
-			host.execOut("\"" + Exe_Mysql_5_7_mysqld + "\" --install", AHost.TEN_MINUTES);
+			host.execElevated("\"" + Exe_Mysql_5_7_mysqld + "\" --install", AHost.TEN_MINUTES);
```

Comment:
I used execOut() on purpose here since the execElevated will return immediately from the pftt2 main thread and open an separated windows to execute the command. The problem is installation of mysqld may not able to complete before executing next command, and next command will fail.

Using execOut() works if pftt2 is invoked with admin privilege, which is required if pftt2 is invoked by pftt.cmd as I remembered.

Reply:
I see. When I was testing this I didn't have eclipse ran with admin privileges so the setup was failing. I tested it manually in CLI and found out it needed admin privileges which is why I switched it to `execElevated`. I'll switch it back for now and see how it behaves. 

Just to make sure, when pftt opens another window to execute the command, does it continue or wait until the installation is complete?

Reply:
My observation is it won't wait the other window to complete. I tried a few ways with elevated executing without luck...